src/xdocs/report_issue.xml

```diff
@@ -40,8 +40,8 @@
     <section name="How to report a bug?">
 
       <p>
-        First and foremost, do not think that your issue is obvious and do not need details.<br/>
-        Even tiny issue need all details
+        First and foremost, do not think that your issue is obvious and doesn't need details.<br/>
+        Even tiny issues need all details
```

Comment:
please revert back, it should be singular.

Reply:
Changed